symfony/client/src/orangehrmAdminPlugin/pages/emailConfiguration/ViewEmailConfiguration.vue

```diff
@@ -0,0 +1,351 @@
+<!--
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+ -->
+
+<template>
+  <div class="orangehrm-background-container">
+    <div class="orangehrm-paper-container">
+      <div class="orangehrm-header-container">
+        <oxd-text tag="h6" class="orangehrm-main-title">
+          Email Configuration
+        </oxd-text>
+      </div>
+      <oxd-divider />
+
+      <oxd-form :loading="isLoading" @submitValid="onSave">
+        <oxd-form-row>
+          <oxd-grid :cols="2" class="orangehrm-full-width-grid">
+            <oxd-grid-item>
+              <oxd-input-field
+                label="Mail Sent As"
+                v-model="emailConfiguration.sentAs"
+                :rules="rules.sentAs"
+                required
+              />
+            </oxd-grid-item>
+            <oxd-grid-item>
+              <oxd-input-group
+                label="Sending Method"
+                :classes="{wrapper: '--status-grouped-field'}"
+                :style="{maxWidth: '12rem'}"
+              >
+                <oxd-input-field
+                  type="radio"
+                  v-model="emailConfiguration.mailType"
+                  optionLabel="Sendmail"
+                  value="sendmail"
+                />
+                <oxd-input-field
+                  type="radio"
+                  v-model="emailConfiguration.mailType"
+                  optionLabel="SMTP"
+                  value="smtp"
+                />
+              </oxd-input-group>
+            </oxd-grid-item>
+          </oxd-grid>
+        </oxd-form-row>
+        <div v-if="emailConfiguration.mailType === 'sendmail'">
+          <oxd-form-row>
```

Comment:
no need for external `div`. give `v-if` directly to `<oxd-form-row>`

Reply:
changed!

Reply:
changed!